packages/react-mutation-mapper/src/store/DefaultMutationMapperDataFetcher.ts

```diff
@@ -314,7 +314,8 @@ export class DefaultMutationMapperDataFetcher
         const structuralQueryVariants: AnnotateStructuralVariantQuery[] = _.uniqBy(
             _.map(
                 queryVariants.filter(
-                    mutation => mutation.mutationType === 'Fusion'
+                    mutation =>
+                        mutation.mutationType?.toUpperCase() === 'FUSION'
```

Comment:
sorta shocking if we don't have an enum for this somewhere?

Reply:
Found one, updated